BUILD.bazel

```diff
@@ -0,0 +1,103 @@
+load(
+    "//ign_bazel:build_defs.bzl",
+    "IGNITION_FEATURES",
+    "IGNITION_VISIBILITY",
+    "cmake_configure_file",
+    "generate_include_header",
+    "ign_config_header",
+    "ign_export_header",
+)
+
+package(
+    default_visibility = IGNITION_VISIBILITY,
+    features = IGNITION_FEATURES,
+)
+
+licenses(["notice"])
+
+exports_files(["LICENSE"])
+
+PROJECT_NAME = "ignition-math"
+
+PROJECT_MAJOR = 6
```

Comment:
```suggestion
PROJECT_MAJOR = 7
```

Reply:
fixed with force-push in https://github.com/ignitionrobotics/ign-math/commit/0523e1abbfcad7836c8af6622668e32fa71b38ce